tests/integration/targets/connection_nsenter/aliases

```diff
@@ -0,0 +1,3 @@
+skip/docker  # this requires unfettered access to the container host
+destructive
+unsupported
```

Comment:
How do you know that the tests do not work in CI?

Reply:
I'm not certain but I'd guess it's related to these requirements from the notes:
```
            This plugin requires the Ansible controller container to be launched in the following way:
            (1) The container image contains the nsenter program;
            (2) The container is launched in privileged mode;
            (3) The container is launched in the host's PID namespace (i.e. C(--pid host)).
```

Reply:
The tests are launching the container, so for the CI instances which are VMs and not containers this should just work. (For the CI instances which are not VMs the tests are alread skipped by the `skip/docker` above.)

Reply:
neat, I bet it's worth a try then!

Reply:
No, without the correct target the tests will never be run.